.travis.yml

```diff
@@ -4,3 +4,10 @@ otp_release:
   - 23.0
   - 22.3
   - 21.3
+  - 20.3
```

Comment:
This file is apparently not used, anymore. Are we testing for OTP 19, in any case?